utility_network/display-content-of-utility-network-container/src/main/java/com/esri/samples/display_content_of_utility_network_container/DisplayContentOfUtilityNetworkContainerController.java

```diff
@@ -82,14 +80,14 @@ public void initialize() {
       mapView.getGraphicsOverlays().add(graphicsOverlay);
 
       // create three new simple line symbols for displaying container view features
-      boundingBoxSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DASH, ColorUtil.colorToArgb(Color.YELLOW), 3);
-      attachmentSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DOT, ColorUtil.colorToArgb(Color.CORNFLOWERBLUE), 3);
-      connectivitySymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DOT, ColorUtil.colorToArgb(Color.RED), 3);
+      boundingBoxSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DASH, Color.YELLOW, 3);
+      attachmentSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DOT, Color.CORNFLOWERBLUE, 3);
+      connectivitySymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DOT, Color.RED, 3);
 
       // set image views for the association and bounding box symbols to display them in the legend
-      attachmentImageView.setImage(attachmentSymbol.createSwatchAsync(0x00000000).get());
-      connectivityImageView.setImage(connectivitySymbol.createSwatchAsync(0x000000000).get());
-      boundingBoxImageView.setImage(boundingBoxSymbol.createSwatchAsync(0x000000000).get());
+      attachmentImageView.setImage(attachmentSymbol.createSwatchAsync(Color.BLACK).get());
+      connectivityImageView.setImage(connectivitySymbol.createSwatchAsync(Color.BLACK).get());
+      boundingBoxImageView.setImage(boundingBoxSymbol.createSwatchAsync(Color.BLACK).get());
```

Comment:
```suggestion
      attachmentImageView.setImage(attachmentSymbol.createSwatchAsync(Color.TRANSPARENT).get());
      connectivityImageView.setImage(connectivitySymbol.createSwatchAsync(Color.TRANSPARENT).get());
      boundingBoxImageView.setImage(boundingBoxSymbol.createSwatchAsync(Color.TRANSPARENT).get());
```
See below for explanation, but these should be transparent rather than black to keep the UI the same